### DISCUSSION TITLE
【求助】这个人是谁？

### DISCUSSION THREAD

**Discussion by LBK-miq** [share121/inter-knot#2766] [其他]
![7d69d60bb289145199aa602539c1f629](https://github.com/user-attachments/assets/97ead55b-372e-4a99-890c-78a3bb212403)

如题，最近本人经常能看到这个形象的画像，还总是听到有许多人讨论这个人，说什么这位和小吉祥草王有很深的关系，小吉祥草王的名声可谓无人不知无人不晓，但是**这位据说和她关系很深的人我怎么一点都没听说过**？这也太奇怪了！特此求助各位绳网大佬，希望哪位大佬能狠狠科普一下，在下不胜感激！

**Comment by OttoApocalypse01** [comment]
这是被我拿走核心的千人律者<img src="https://bbs-static.miyoushe.com/static/2023/05/31/9834f2be541e2cb81b8a4cbf5160b2b0_6075466775997410414.png" alt="罗刹 治疗" width="75" >

**Comment by zhixuan45** [comment]
纳尼？本地人！

**Comment by dnmdngcsb** [comment]
我认为你可以去问问某位黄毛旅行者，可以去最近的冒险家协会挂个委托，记得报酬给1600原石以上

**Reply by LBK-miq** [reply]
别提了，上次碰到的那位黄毛开口就是1600纠缠之缘起步，给我都吓到了

**Comment by lllliufeng** [comment]
原来虚空终端可以连绳网啊（

**Comment by gitmolin** [comment]
绳网能连上虚空终端，那连上联觉信标也不奇怪吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
